res/controllers/Hercules DJ Console 4-Mx.midi.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version='1.0' encoding='utf-8'?>
 <MixxxControllerPreset mixxxVersion="2.0.0+" schemaVersion="1">
```

Comment:
Currently it is compatible with 2.0. I've left there some comments for 2.1 compatibility but aside of that it's fine. I've tested it with 2.0 release, although not extensively. I use basically a 2.1 alpha. Just labeled (2.1) so that it can be differentiated from the older one.